docs/testing/stores.md

```diff
@@ -113,7 +130,14 @@ describe('PetStore', () => {
   // lets use this inaccessible method to show how we can test
   // non static methods if we desire/need to
   it('rounds money to 2 decimal places', function() {
-    var unwrappedStore = new UnwrappedPetStore();
+    class TestPetStore extends UnwrappedPetStore {
+      constructor() {
+        // we extend and override the constructor because
```

Comment:
Ah was not aware of that. We could just promote blackbox testing.  Any "private" functions should have affects that can be measured elsewhere in the store.  I know at least in the Ruby community, the testing of private methods is not endorsed.

So I am wondering if there is a case where we are unable to test the store just using its publicly accessible functions.  I actually think this might be rare given the purpose of stores.